content/index.md

```diff
@@ -0,0 +1,26 @@
+# quartz-typst
```

Comment:
can you remove this file?

Reply:
lets add docs for typst support before merging